javascript/javascript1/week2/fullnameFunction.js

```diff
@@ -0,0 +1,11 @@
+function getFullname(firstname, surname) {
+  return firstname + surname;
+}
```

Comment:
Remove this. You don't have to declare the same function twice. Already you have implemented the function that does the intended

Reply:
Okay!